common/scripts/babel-conditional-preprocess.js

```diff
@@ -69,6 +69,10 @@ function Handle(path, annotations) {
         }
         if (removed) {
           comment.ignore = true;
+          // Comment will be inherit to next line even it is set to 'ignore'
+          // this will cause some unexpected removals of code
+          // clear the comment to ensure nothing gets wrong
+          comment.value = '';
```

Comment:
I'd recommend describing an example of what incorrect removal happens without this setting. It'll be very hard for a reader in the future to understand the implications of removing this value.